dist/pathfora.js

```diff
@@ -2222,8 +2239,12 @@
         pathforaDataObject.closedWidgets.push(params);
         break;
       case 'confirm':
-        params['pf-widget-action'] = !!widget.confirmAction && widget.confirmAction.name || 'default confirm';
-        pathforaDataObject.completedActions.push(params);
+        if (htmlElement && utils.hasClass(htmlElement, 'pf-content-unit')) {
+          params['pf-widget-action'] = 'content recommendation';
```

Comment:
Yeah, that's the only reason I put spaces in it, cause the others have them. :/ Kinda weird, but at least it's consistent.